ios/Modules/A11yModule/RCA11yModule.swift

```diff
@@ -44,8 +44,10 @@ class RCA11yModule : RCTEventEmitter, KeyboardFocusManager, A11yOrderManager {
     
     @objc
     func announceForAccessibility(_ announcement: String) -> Void {
-        let message: NSAttributedString = NSAttributedString(string: announcement, attributes: [.accessibilitySpeechQueueAnnouncement: true])
-        UIAccessibility.post(notification: .announcement, argument: message)
+        if #available(iOS 11.0, *) {
```

Comment:
like for this. Anyway, do we need it? This logic is not implemented in RN .67 by default

Reply:
I mean checking for IOS version

Reply:
I had some problems after updating xcode and macos when I build the app. I don't know why but right know it is required. Maybe some fixes from xcode maybe some configuration was broken, who knows. I don't think that we will have a problem with it